fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/domain/ConfigurationDomainServiceJpa.java

```diff
@@ -319,11 +318,12 @@ public Long getDailyTPTLimit() {
         return property.getValue();
     }
 
+    @CacheEvict(value = "configByName", key = "T(org.apache.fineract.infrastructure.core.service.ThreadLocalContextUtil).getTenant().getTenantIdentifier().concat(#propertyName)")
     @Override
     public void removeGlobalConfigurationPropertyDataFromCache(final String propertyName) {
-        String identifier = ThreadLocalContextUtil.getTenant().getTenantIdentifier();
-        String key = identifier + "_" + propertyName;
-        configurations.remove(key);
+        if (log.isDebugEnabled()) {
```

Comment:
No need for the if, log.debug will just do the job. the isDebugEnabled check should be used when the value being logged is calculated and potentially expensive (for example making a DB call or something).